packages/button/src/Button.js

```diff
@@ -0,0 +1,10 @@
+import BaseButton from './BaseButton'
```

Comment:
With the default Button export, do we want to allow an `outline` prop to be passed in to toggle which kind of button is being used, or are we set on requiring an explicit import of OutlineButton?